packages/calling-component-bindings/src/utils/participantListSelectorUtils.ts

```diff
@@ -9,41 +12,31 @@ import { CallParticipantListParticipant } from '@internal/react-components';
  * @private
  */
 export const memoizedConvertAllremoteParticipants = memoizeFnAll(
-  (
-    userId: string,
-    displayName: string | undefined,
-    state: 'Idle' | 'Connecting' | 'Ringing' | 'Connected' | 'Hold' | 'InLobby' | 'EarlyMedia' | 'Disconnected',
-    isMuted: boolean,
-    isScreenSharing: boolean,
-    isSpeaking: boolean
-  ): CallParticipantListParticipant => {
-    return convertRemoteParticipantToParticipantListParticipant(
-      userId,
-      displayName,
-      state,
-      isMuted,
-      isScreenSharing,
-      isSpeaking
-    );
+  (data: {
```

Comment:
Similar to what James said above. The first function param is used as the key and this won't be memoized since you are passing an object.